code/__DEFINES/~whitesands_defines/metacoin.dm

```diff
@@ -7,8 +7,8 @@
 /// Rewarded when you escape on the shuttle
 #define METACOIN_ESCAPE_REWARD(is_speed_round, round_duration) is_speed_round ? 100 : max(min(round(100 * (round_duration / 72000)), 100), 100) // VOID CREW EDIT
 /// Rewarded when you survive the round
-#define METACOIN_SURVIVE_REWARD(is_speed_round, round_duration) is_speed_round ? 50 : max(min(round(50 * (round_duration / 72000)), 50), 50) // VOID CREW EDIT
+#define METACOIN_SURVIVE_REWARD(is_speed_round, round_duration) is_speed_round ? 150 : max(min(round(50 * (round_duration / 72000)), 50), 50) // VOID CREW EDIT
```

Comment:
Do rounds on voidcrew really last 5 minutes that often that the amount of metacoins has to be buffed for when it does?